x-pack/test/cloud_security_posture_functional/config.ts

```diff
@@ -26,8 +25,21 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
       ...xpackFunctionalConfig.get('kbnTestServer'),
       serverArgs: [
         ...xpackFunctionalConfig.get('kbnTestServer.serverArgs'),
-        ...getPreConfiguredFleetPackages(),
-        ...getPreConfiguredAgentPolicies(),
+        /**
+         * Package version is fixed (not latest) so FTR won't suddenly break when package is changed.
+         *
+         * test a new package:
+         * 1. build the package and start the registry with elastic-package and uncomment the 'registryUrl' flag below
+         * 2. locally checkout the kibana version that matches the new package
+         * 3. update the package version below to use the new package version
+         * 4. run tests with NODE_EXTRA_CA_CERTS pointing to the elastic-package certificate
+         * 5. when test pass:
+         *   1. release a new package to EPR
+         *   2. merge the updated version number change to kibana
+         */
+        `--xpack.fleet.packages.0.name=cloud_security_posture`,
+        `--xpack.fleet.packages.0.version=1.0.8`,
```

Comment:
notable changes: 

1 - not using fleet agent policies flag, since our UI only requires and installed package and indexed findings, we don't need that for current tests, will add when needed (was used when we had benchmark tests)
2 - using `1.0.8` because newer version for `8.7` include `posture/deployment` vars which fail the initial fleet setup. test would still pass, but as we're going to change those vars to be non-required anyway, i prefer bumping the package used in FTR when we do that.